src/test/java/plugins/AntPluginTest.java

```diff
@@ -54,24 +94,96 @@ public void autoInstallMultipleAnt() {
         );
     }
 
-    @Test @Native("ant")
+    @Test
     public void locallyInstalledAnt() {
+        useCustomAgent();
```

Comment:
```suggestion

        final String NATIVE_ANT_NAME = "native_ant"
        useCustomAgent();
```

Reply:
`NATIVE_ANT_NAME` is used in several methods and tests, I thought not to multiply the declaration